sentry/build.gradle.kts

```diff
@@ -19,6 +19,7 @@ tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach
 dependencies {
     // Envelopes require JSON. Until a parse is done without GSON, we'll depend on it explicitly here
     implementation(Config.Libs.gson)
+    implementation(Config.Libs.apacheHttpClient)
```

Comment:
I think this is very problematic for Android, first because there's a dependency that is not used, and even worse, there are a bunch of transitive dependencies.
I'd even say that they are not Android compatible.